service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/spring/KeycloakServiceImpl.java

```diff
@@ -334,8 +340,8 @@ private ClientRepresentation isClientExists(PushApplication pushApp) {
 		return isClientExists(getClientd(pushApp));
 	}
 
-	public static String getClientd(PushApplication pushApp) {
-		return CLIENT_PREFIX + pushApp.getName().toLowerCase();
+	public static String getClientd(PushApplication pushAppName) {
+		return CLIENT_PREFIX + pushAppName.getName().toLowerCase();
 	}
```

Comment:
1. Feels like a typo error - perhaps `getClientId` ?
2. Why rename the parameter ? It is `pushApp` and not `pushAppName` ...